dropfile_formats/CHAIN.TXT

```diff
@@ -0,0 +1,31 @@
+{UI}
+{UN}
+{UR}
+?
+{UA}
+M
+{AP}
+{getCurrentDateMMDDYY}
+{SW}
+{SH}
+{LL}
+0
+0
+1
+0
+99999
+C:\DATA
```

Comment:
I wonder if we'll need to allow these to be configured. I imagine a WWIV door running under e.g. DOSEMU, where `D:\SOMETHING\DATA` is where it wants to perform I/O. I'm not sure though...

Reply:
They absolutely will need to be - I didn't have anything to set them to yet, so left them for now. Before this gets merged though hopefully will be able to set them to what we need for either existing door system or the new one. I just didn't want to wait for that for you to get a chance to check out the overall direction etc